src/System.ValueTuple/src/System/Numerics/Hashing/Hash.cs

```diff
@@ -0,0 +1,101 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+
+namespace System.Numerics.Hashing
+{
+    internal static class Hash
```

Comment:
I was thinking this whole PR would boil down to just having the new HashHelpers.Combine(int, int) method in its own file, and either deleting the existing ValueTuple.CombineHashCodes(int, int) method and replacing its call sites with calls to HashHelpers.Combine(int, int), or simply having ValueTuple.CombineHashCodes(int, int) call HashHelpers.Combine(int, int).